package.json

```diff
@@ -21,31 +21,35 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint .",
-    "test": "mocha --timeout 15000 --require test/init.js",
+    "test": "mocha --timeout 180000  test/*.test.js node_modules/juggler-v3/test.js node_modules/juggler-v4/test.js",
     "posttest": "npm run lint"
   },
   "dependencies": {
-    "async": "^1.5.0",
+    "async": "^3.1.0",
     "debug": "^2.2.0",
-    "loopback-connector": "^4.0.0",
-    "odbc": "^2.1.2",
-    "strong-globalize": "^2.8.2"
+    "loopback-connector": "^4.9.0",
+    "moment": "^2.24.0",
+    "moment-timezone": "^0.5.21",
+    "odbc": "^2.2.2",
+    "strong-globalize": "^5.0.2"
   },
   "devDependencies": {
     "bluebird": "^2.9.9",
     "eslint": "^5.15.3",
     "eslint-config-airbnb-base": "^13.1.0",
     "eslint-config-loopback": "^13.0.0",
     "eslint-plugin-import": "^2.16.0",
-    "loopback-datasource-juggler": "^2.43.0",
+    "juggler-v3": "file:./deps/juggler-v3",
+    "juggler-v4": "file:./deps/juggler-v4",
+    "loopback-datasource-juggler": "^4.13.0",
     "mocha": "^6.1.4",
     "rc": "^1.1.5",
     "should": "^8.1.1",
     "sinon": "^1.17.2"
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/strongloop/loopback-connector-ibmi.git"
+    "url": "git://github.com/https://github.com/strongloop/loopback-connector-ibmi.git"
```

Comment:
This looks like a typo to me, you have two protocols `git://` and `https://`. Let's keep the https protocol only.